sdk/core/Cargo.toml

```diff
@@ -13,24 +13,26 @@ categories = ["api-bindings"]
 edition = "2018"
 
 [dependencies]
+async-trait = "0.1"
+bytes = "1.0"
 chrono = "0.4"
-http = "0.2"
+clap = "2.33.3"
```

Comment:
No, please remove `clap`.

Reply:
It is just two macros that can be redefined:
https://github.com/clap-rs/clap/blob/master/src/macros.rs